src/Catalyst.Core.Lib.Tests/UnitTests/Extensions/ProtobufExtensionsTests.cs

```diff
@@ -0,0 +1,92 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Linq;
+using Catalyst.Core.Lib.Extensions;
+using Catalyst.Core.Lib.IO.Messaging.Correlation;
+using Catalyst.Core.Lib.Util;
+using Catalyst.Protocol.IPPN;
+using Catalyst.Protocol.Peer;
+using Catalyst.Protocol.Rpc.Node;
+using Catalyst.TestUtils;
+using FluentAssertions;
+using Google.Protobuf;
+using Xunit;
+
+namespace Catalyst.Core.Lib.Tests.UnitTests.Extensions
+{
+    public class ProtobufExtensionsTests
+    {
+        [Fact]
+        public static void ToAnySigned_should_happen_new_guid_to_request_if_not_specified()
+        {
+            //this ensures we won't get Guid.Empty and then a risk of mismatch;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4318201)